### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by madalinazanficu**
Added translations for some screen inside Observation flow:
- index.tsx (observation) - Select Polling Station (bottom sheet)
                                        - Polling Station General (excepting Time Select)
                                        -  Form overview [formId].tsx
                                        - TimeSelect
                                        - [question].id
                                        - Change Language Dialog
                                        - EditNote Sheet Content
                                        - Add Note Sheet Content
                                        - FormCard
                                        - Select component
TODO:
- TimeSelect inside [questionId].tsx

### CODE REVIEW

**Comment by luciatugui** [comment]
please make sure all romanian translations have diacritical marks

File: mobile/assets/locales/ro/translations.json [hunk 1]
```diff
@@ -101,21 +101,28 @@
...
     }
   },
   "bottom_sheets": {
     "observations": {
+      "title": "Sectile mele de votare",
+      "paragraph": "Puteți comuta între secțiile de votare dacă doriți să revizuiți răspunsurile din formulare sau informațiile despre secțiile de votare.",
       "actions": {
-        "clear_form": "Ștergeți formularul (ștergeți toate răspunsurile)"
+        "clear_form": "Ștergeți formularul (ștergeți toate răspunsurile)",
+        "add_station": "Adăugați o nouă secție de votare.",
```
Comment (luciatugui) [inline mobile/assets/locales/ro/translations.json:124]: Please remove the '.' from the end as this text is displayed inside a button

File: mobile/assets/locales/ro/translations.json [hunk 2]
```diff
@@ -101,21 +101,28 @@
...
     "header": {
       "title": "Informații despre secția de votare"
+    },
+    "actions": {
+      "submit": "Submiteti raspunsurile"
     }
   },
   "bottom_sheets": {
     "observations": {
+      "title": "Sectile mele de votare",
```
Comment (luciatugui) [inline mobile/assets/locales/ro/translations.json:120]: diacritical marks and check spelling

Comment (madalinazanficu) [reply mobile/assets/locales/ro/translations.json]: done

File: mobile/assets/locales/ro/translations.json (hunk 1 above)
Comment (madalinazanficu) [reply mobile/assets/locales/ro/translations.json]: done

**Review by luciatugui — APPROVED** [review]

**Pull request merged**